File: benchmark/results/ts-monolith-sonnet-script/run-003/b-baseline/output/src/orders/fulfillment/value-objects/ShipmentRoute.ts
import { Id } from '../../../shared/value-objects/Id';
import { Timestamp } from '../../../shared/value-objects/Timestamp';

/** ShipmentRoute value object */
export interface ShipmentRoute {
  readonly id: Id;
  readonly createdAt: Timestamp;
  readonly updatedAt: Timestamp;
  readonly name: string;
  readonly description: string;
}

File: benchmark/results/java-invocation-script/run-004/a-mcp/output/shipping/value_objects/ShipmentRoute.java
package com.metaengine.demo.shipping.value_objects;

import java.time.OffsetDateTime;

// ShipmentRoute value object.

public class ShipmentRoute {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public ShipmentRoute(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}
